frontend/projects/valtimo/case/src/lib/case-inspection/tabs/process-detail.component.scss

```
@@ -0,0 +1,119 @@
+
+    &:hover {
+      color: var(--cds-link-primary-hover, #0043ce);
+    }
+
+    span {
+      margin-left: 0.25rem;
+    }
+  }
+
+  &__variable-value {
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)